portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -87,9 +89,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // If statement determines whether the POST request is
     // from limiting a certain number of comments to display
     // or from leaving a new comment
+    UserService userService = UserServiceFactory.getUserService();
     if (request.getParameter("numComment") != null) {
       numComment = Integer.parseInt(request.getParameter("numComment"));
     } else {
+      if (!userService.isUserLoggedIn()) {
```

Comment:
Does this mean user can enter number of comments to display while logged out?

Reply:
Yes, users should be able to see comments and choose the number they want to display even if they're not logged in.